app/src/main/res/layout/welcome.xml

```diff
@@ -33,16 +33,7 @@
         android:layout_height="wrap_content"
         android:layout_marginEnd="@dimen/home_welcome_text_view_margin_end"
         android:fontFamily="sans-serif"
-        android:text="@{viewModel.greeting}"
-        android:textColor="@color/component_color_shared_primary_text_color"
-        android:textSize="24sp" />
-
-      <TextView
-        android:id="@+id/profile_name_text_view"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:fontFamily="sans-serif"
-        android:text="@{viewModel.computeProfileNameText()}"
+        android:text="@{viewModel.welcomeText}"
```

Comment:
Isn't this supposed to be a function call?

Please upload screenshots of the changes for LTR and RTL.

Reply:
I'm using a Windows machine and my Android studio build tool is Gradle. As app language option is not showing when app built with gradle.
That's why I can't change the language to check for the RTL.
Any suggestions?

Reply:
If you change the language of your device/emulator to arabic, it will assume a RTL locale, and you will notice that the app displays in RTL as well.